core/jazz_email/package.json

```diff
@@ -12,6 +12,7 @@
     "jshint": "jshint --exclude ./node_modules ."
   },
   "dependencies": {
+    "aws-sdk": "^2.380.0",
```

Comment:
Why do we need this? In actual AWS environment, this is injected automatically (we don't need to package it during deployment explicitly). If you need this for running unit tests, you can add it to devDependencies.

Reply:
yes it was for unit tests. updated